pos_lot_barcode/static/src/js/edit_list_popup.js

```diff
@@ -0,0 +1,30 @@
+/*
+    Copyright 2022 Camptocamp SA
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+*/
+odoo.define("pos_lot_barcode.EditListPopup", function (require) {
+    "use strict";
+    
+    const EditListPopup = require("point_of_sale.EditListPopup");
+    const Registries = require("point_of_sale.Registries");
+    const { onChangeOrder, useBarcodeReader } = require('point_of_sale.custom_hooks');
```

Comment:
```suggestion
    const { useBarcodeReader } = require('point_of_sale.custom_hooks');
```
`onChangeOrder` isn't used